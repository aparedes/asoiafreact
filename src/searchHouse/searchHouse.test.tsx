import { render } from '../../tests/test-utils';
import SearchHouse from './searchHouse';

test('Prints a list o<f houses', () => {
  const { container } = render(<SearchHouse />);

  expect(container).toMatchSnapshot();
});
