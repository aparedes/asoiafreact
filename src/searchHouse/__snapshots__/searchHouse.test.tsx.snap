// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prints a list o<f houses 1`] = `
<div>
  <input
    class="search"
    placeholder="Search house"
  />
</div>
`;
